docker/pgbouncer/entrypoint.sh

```
@@ -53,41 +53,30 @@ parse_url() {
 
 # Grabs variables set by `parse_url` and adds them to the userlist if not already set in there.
 generate_userlist_if_needed() {
-  if [ -n "${DB_USER}" ] && [ -n "${DB_PASSWORD}" ] && [ -e "${_AUTH_FILE}" ] && ! grep -q "^\"${DB_USER}\"" "${_AUTH_FILE}"; then
+  if [ -n "${DB_USER}" -a -n "${DB_PASSWORD}" -a -e "${_AUTH_FILE}" ] && ! grep -q "^\"${DB_USER}\"" "${_AUTH_FILE}"; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)